src/test/java/org/assertj/core/api/optional/OptionalAssert_get_Test.java

```diff
@@ -13,37 +13,87 @@
 package org.assertj.core.api.optional;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.assertj.core.api.BDDAssertions.then;
 import static org.assertj.core.error.OptionalShouldBePresent.shouldBePresent;
+import static org.assertj.core.presentation.UnicodeRepresentation.UNICODE_REPRESENTATION;
+import static org.assertj.core.test.AlwaysEqualComparator.ALWAY_EQUALS;
+import static org.assertj.core.util.AssertionsUtil.expectAssertionError;
 import static org.assertj.core.util.FailureMessages.actualIsNull;
 
 import java.util.Optional;
 
-import org.assertj.core.api.BaseTest;
-import org.assertj.core.data.TolkienCharacter;
-import org.assertj.core.data.TolkienCharacter.Race;
+import org.assertj.core.api.AbstractAssert;
+import org.assertj.core.api.AbstractObjectAssert;
+import org.assertj.core.api.OptionalAssert;
+import org.assertj.core.util.introspection.PropertyOrFieldSupport;
+import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
 /**
+ * Tests for <code>{@link OptionalAssert#get()}</code>.
+ *
  * @author Filip Hrisafov
  */
-public class OptionalAssert_get_Test extends BaseTest {
+@DisplayName("OptionalAssert get")
+class OptionalAssert_get_Test {
+
+  private final Optional<String> optional = Optional.of("Frodo");
+
+  @Test
+  void should_fail_if_optional_is_null() {
+    // GIVEN
+    Optional<String> optional = null;
+    // WHEN
+    AssertionError assertionError = expectAssertionError(() -> assertThat(optional).get());
+    // THEN
+    then(assertionError).hasMessage(actualIsNull());
+  }
 
   @Test
-  public void should_fail_when_optional_is_null() {
-    assertThatExceptionOfType(AssertionError.class).isThrownBy(() -> assertThat((Optional<String>) null).get())
-                                                   .withMessage(actualIsNull());
+  void should_fail_if_optional_is_empty() {
+    // GIVEN
+    Optional<String> optional = Optional.empty();
+    // WHEN
+    AssertionError assertionError = expectAssertionError(() -> assertThat(optional).get());
+    // THEN
+    then(assertionError).hasMessage(shouldBePresent(optional).create());
   }
 
   @Test
-  public void should_fail_when_optional_is_empty() {
-    assertThatExceptionOfType(AssertionError.class).isThrownBy(() -> assertThat(Optional.<String>empty()).get())
-                                                   .withMessage(shouldBePresent(Optional.<String>empty()).create());
+  void should_pass_if_optional_contains_a_value() {
+    // WHEN
+    AbstractObjectAssert<?, String> result = assertThat(optional).get();
+    // THEN
+    result.isEqualTo("Frodo");
   }
 
   @Test
-  public void should_pass_when_optional_contains_a_value() {
-    TolkienCharacter frodo = TolkienCharacter.of("Frodo", 33, Race.HOBBIT);
-    assertThat(Optional.of(frodo)).get().hasNoNullFieldsOrProperties();
+  void should_keep_existing_description() {
```

Comment:
can be removed because the description is part of `AssertionInfo` that is tested in `should_keep_existing_assertion_state`

Reply:
Dumb copy-paste mistake! 🤦‍♂